v1/card/card.gunk

```diff
@@ -90,6 +87,16 @@ type Card struct {
 	AccessStatus types.CardAccessStatus `pb:"9" json:"access_status"`
 }
 
+// CardAttribute hold all detail about card attribute
+type CardAttribute struct {
+	// ID is a unique identifier of the attribute
```

Comment:
either remove this comment, or put comments on all of the other fields @nkristianto